### PR TITLE
YIQ Colors

### PR DESCRIPTION
**Pull request by BladeBronson**
This is a subset of my last Wikia hack, Better Colors. This uses YIQ color theory to calculate the best contrasting color (white or black) for a given background color.

### CODE REVIEW

File: skins/shared/_yiq-color-contrast.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// From: https://github.com/timhettler/compass-yiq-color-contrast
```
Comment (lizlux) [inline skins/shared/_yiq-color-contrast.scss]: Can this file go into a lib folder?

**Comment by lizlux** [comment]
tagging @inez and @hakubo as content team tech leads

**Comment by hakubo** [comment]
Do we need to install something to use that?

**Comment by hakubo** [comment]
@inez do you know more about this?

It looks like a nice idea, but I don't know much about it, could you help me here?

**Comment by inez** [comment]
@hakubo Yes. This is @BladeBronson's implementation of his hack (from Hackathon 2014) that improves contrast of text color + background color (+some other) in certain cases - it was well explained in hackathon video, but I can't get access to it now (I think this is because it was posted to Google+, but @BladeBronson does not have Wikia's account there anymore).
This is not a high priority, I will look closer into it after I'm done with upstream sync.

**Comment by inez** [comment]
@lizlux Feel free to re-assign to me if you like.

**Comment by hakubo** [comment]
Thanks!

**Comment by dianafa** [comment]
@BladeBronson @inez is this PR still valid?

**Comment by inez** [comment]
This is entirely up to @BladeBronson

**Comment by hakubo** [comment]
I see no interest from anyone to work on it
and it sounds to me like it could have big impact on how we render text on wikis
I do believe that this looks better, but I see no hands to go and test it.
Will close it for now as we spend already to much time on it.

**Pull request closed without merging**